e2e/integration/filters/clearFilters.spec.js

```diff
@@ -52,6 +50,30 @@ describe('Clear Filters',() => {
         .should('not.contain','?');
     });
 
+    it('Clear all refinements should not reset search, per page, and sorting', () => {
```

Comment:
Added a test to ensure that search, per page, and sorting are not reset when clearing active filters